torch/_tensor_docs.py

```diff
@@ -1741,25 +1767,24 @@ def add_docstr_all(method, docstr):
 
 add_docstr_all('sparse_mask',
                r"""
-sparse_mask(input, mask) -> Tensor
+sparse_mask(mask) -> Tensor
 
-Returns a new SparseTensor with values from Tensor :attr:`input` filtered
-by indices of :attr:`mask` and values are ignored. :attr:`input` and :attr:`mask`
-must have the same shape.
+Returns a new :ref:`sparse tensor <sparse-docs>` with values from
+:attr:`self` tensor filtered by indices of :attr:`mask`. The values of
+:attr:`mask` sparse tensor are ignored. :attr:`self` and :attr:`mask`
+tensors must have the same shape.
 
 Args:
-    input (Tensor): an input Tensor
-    mask (SparseTensor): a SparseTensor which we filter :attr:`input` based on its indices
+    mask (Tensor): a sparse tensor which we filter :attr:`self` tensor based on its indices
```

Comment:
We should avoid use of language like "we." 

"the sparse tensor whose indices are used as a filter"

Maybe that could also be improved to clarify how the filter works?

Reply:
updated.